### PR TITLE
Adding Riquelme 2022 to dashboard, attempt 2

### PR DESCRIPTION
**Pull request by willbradshaw**
NB: First attempt had issues with merge conflicts; ended up reverting, merging the last pull request, then re-attempting.

### CODE REVIEW

**Comment by willbradshaw** [comment]
Note: I'm getting a warning: "File dashboard/data.js is 50.49 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB"

File: dashboard/human_virus_sample_counts.json
Comment (jeffkaufman) [inline dashboard/human_virus_sample_counts.json:1]: Whoops!  The dashboard generation script removed all entries from `human_virus_sample_counts.json` except those from your study.

Looking back, what happened is:

* Yesterday I made a change (903ae4af86c16d0a0aa11f9ee6d8bca52f8092ad) where the `hvreads` files now include quality scores.
* I forgot to update `dashboard/prepare-dashboard-data.py` to handle this new format
* Only your `hvreads` files (which were created at the same time I was making my change and regenerating old data) had the old format
* Only your samples were eligible for inclusion in `human_virus_sample_counts.json`

So now:

* I'm fixing `dashboard/prepare-dashboard-data.py` to handle this new format
* I'm regenerating your `hvreads` files

Once that's done you should be able to do a `rm -r dashboard/hvreads` and then re-run `dashboard/prepare-dashboard-data.sh` successfully.

**Review by jeffkaufman — CHANGES_REQUESTED** [review]

File: dashboard/human_virus_sample_counts.json
Comment (jeffkaufman) [reply dashboard/human_virus_sample_counts.json:1]: > I'm fixing dashboard/prepare-dashboard-data.py to handle this new format

Done: 71287953a24bf9a88f6296f8da6a68e2b1b16f0b

> I'm regenerating your hvreads files

Done

**Comment by willbradshaw** [comment]
@jeffkaufman I merged the changes from main into this branch and regenerated the dashboard files as requested.

**Review by jeffkaufman — APPROVED** [review]

**Comment by jeffkaufman** [comment]
Looks good, thanks!

**Pull request merged**